client/my-sites/site-settings/jetpack-module-toggle.jsx

```diff
@@ -45,18 +45,19 @@ class JetpackModuleToggle extends Component {
 
 	handleChange = () => {
 		if ( ! this.props.checked ) {
-			this.recordTracksEvent( 'calypso_jetpack_module_toggle_activated' );
+			this.recordTracksEvent( 'calypso_jetpack_module_toggle', 'on' );
 			this.props.activateModule( this.props.siteId, this.props.moduleSlug );
 		} else {
-			this.recordTracksEvent( 'calypso_jetpack_module_toggle_deactivated' );
```

Comment:
Deprecate: 1186-gh-tracks-events-validation-status

Reply:
yes, it was my intention to mark those after merging these changes